packages/atomic/src/components/atomic-breadbox/atomic-breadbox.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import defaultStory from '../../../.storybook/default-story';
+import BreadboxDoc from './atomic-breadbox.mdx';
+import {html} from 'lit-html';
+
+const {defaultModuleExport, exportedStory} = defaultStory(
+  'Atomic/Breadbox',
+  'atomic-breadbox',
+  {},
+  BreadboxDoc,
+  {
+    additionalMarkup: () =>
+      html`
```

Comment:
Didn't know this kind of syntax existed in TS/JS. I've only seen that as "macros" in other languages.